src/components/elements/back-to-link.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from "next/link"
+import React, {ElementType, HtmlHTMLAttributes} from "react"
+import {ArrowLongLeftIcon} from "@heroicons/react/24/outline"
+import {twMerge} from "tailwind-merge"
+
+type BackToLinkProps = HtmlHTMLAttributes<HTMLDivElement> & {
+  isArticle?: boolean
+  linkClasses?: string
+  href: string
+  title?: string
+}
+
+const BackToLink = ({isArticle, children, linkClasses, href, title = "Books", ...props}: BackToLinkProps) => {
+  const Wrapper: ElementType = isArticle ? "article" : "div"
+  return (
+    <>
+      <Wrapper {...props} className={twMerge("order-last", props.className)}>
+        {children}
+      </Wrapper>
+      <Link
+        href={href}
```

Comment:
I think what would make this easier to use is remove the fragment. Then the `<Wrapper>` will contain the `flex flex-row` and the `<Link>` will contain a `order-first` and be inside the `<Wrapper>`.

That if we need to change the order, we still only do it in one file instead of the places where this component is used.